test/packages/bad_dynamic_template/changelog.yml

```diff
@@ -0,0 +1,5 @@
+- version: 0.1.2
+  changes:
+    - description: initial release
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/131
```

Comment:
we used to link it to spec's PR. In this case: https://github.com/elastic/package-spec/pull/188

Reply:
Done.